### PR TITLE
chore: Fix Initial Accessibility Testing Issues

### PR DESCRIPTION
**Pull request by shafqat-muneer**
[LEARNER-10022](https://2u-internal.atlassian.net/browse/LEARNER-10022): iOS - Fix Initial Accessibility Testing Issues

### CODE REVIEW

File: Core/Core/en.lproj/Localizable.strings [hunk 1]
```diff
@@ -86,7 +86,8 @@
 "SETTINGS.DOWNLOAD_QUALITY_720_DESCRIPTION" = "Best quality";
 
 "DONE" = "Done";
-"VIEW " = "View";
+"VIEW" = "View";
+"BACK" = "back";
```
Comment (saeedbashir) [inline Core/Core/en.lproj/Localizable.strings:90]: how about using `Back` instead of `back`?

Comment (saeedbashir) [inline Core/Core/en.lproj/Localizable.strings:90]: how about using `Back` instead of `back`?

**Review by saeedbashir — DISMISSED** [review]
LGTM 👍

**Review by rnr — APPROVED** [review]

**Comment by shafqat-muneer** [comment]
We're closing this PR and moving these changes to our own fork, as they are critical for us. After our release, we hope to find a way to contribute these changes back, thanks!

**Pull request closed without merging**